test/integration/api/images.bats

```diff
@@ -68,3 +68,15 @@ function teardown() {
 	[ "${#lines[@]}" -eq 2 ]
 	[[ "${lines[1]}" == *"testimage"* ]]
 }
+
+@test "docker images - after build on engine side" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker -H ${HOSTS[0]} build --no-cache -t testimage:latest $TESTDATA/build
+
+	run docker_swarm images testimage
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 2 ]
```

Comment:
It will only show testimage, such as(2 = header + image)
```
root@SZX1000041894:/home# docker images centos
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
centos              latest              14dab3d40372        2 weeks ago         194.7 MB
```

But this PR still have problem(CI is not OK), It will continue working on this.